.github/workflows/test-repeater.yml

```diff
@@ -36,7 +36,7 @@ on:
         type: choice
         options:
           - 'ubuntu-latest'
-          - 'macos-latest'
+          - 'macos-12'
```

Comment:
This needs to stay macos-12 because we may need to test for flakiness with py-38

Reply:
```suggestion
          - 'macos-12'  # macos-latest doesn't support py38 or py39
```